informative_sampling/Sample[] samples
informative_sampling/Bounds bounds
informative_sampling/Noise noise
---
informative_sampling/BeliefModelParameters params
